how-tos/focus_for_android.md

```diff
@@ -0,0 +1,19 @@
+# Manually sign Focus for Android APKs
+
+There are 2 APKs: one is Focus, the other for the german-speaking population: Klar.
+
+## Create signatures
+1. `ssh signing4.srv.releng.scl3.mozilla.com`
+1. Download unsigned APKs
+1. `jarsigner -keystore /builds/signing/rel-key-signing-server/secrets/focus-jar app-focus-webkit-release-unsigned.apk focus`. Repeat for Klar: only change the name of the APK, the certificate alias (`focus`) remains identical.
```

Comment:
You'll be prompted for the focus passphrase which is stored in the releng private repo.

Reply:
You'll also get a warning "Warning: 
No -tsa or -tsacert is provided and this jar is not timestamped. Without a timestamp, users may not be able to validate this jar after the signer certificate's expiration date (2044-10-25) or after any future revocation date."

Reply:
nice catch @kmoir !